From 3cc52c1d9e674f591f3599a8002fd3636e28c0d8 Mon Sep 17 00:00:00 2001
From: Niko Tyni <ntyni@debian.org>
Date: Fri, 6 Dec 2013 19:52:09 +0200
Subject: Fix spelling of IPC_CREAT in IPC-SysV documentation

Change picked from IPC-SysV-2.04.

Origin: upstream
Bug: https://rt.cpan.org/Public/Bug/Display.html?id=86736
Bug-Debian: http://bugs.debian.org/730558
Patch-Name: fixes/IPC-SysV-spelling.diff
---
 cpan/IPC-SysV/lib/IPC/SysV.pm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cpan/IPC-SysV/lib/IPC/SysV.pm b/cpan/IPC-SysV/lib/IPC/SysV.pm
index 247d199..06723d2 100644
--- a/cpan/IPC-SysV/lib/IPC/SysV.pm
+++ b/cpan/IPC-SysV/lib/IPC/SysV.pm
@@ -113,7 +113,7 @@ C<IPC::SysV> defines and conditionally exports all the constants
 defined in your system include files which are needed by the SysV
 IPC calls.  Common ones include
 
-  IPC_CREATE IPC_EXCL IPC_NOWAIT IPC_PRIVATE IPC_RMID IPC_SET IPC_STAT
+  IPC_CREAT IPC_EXCL IPC_NOWAIT IPC_PRIVATE IPC_RMID IPC_SET IPC_STAT
   GETVAL SETVAL GETPID GETNCNT GETZCNT GETALL SETALL
   SEM_A SEM_R SEM_UNDO
   SHM_RDONLY SHM_RND SHMLBA
